frontend/app/src/screens/LoanScreen/LoanScreen.tsx

```diff
@@ -75,6 +75,67 @@ const TABS = [
   },
 ];
 
+function TroveHistoryLinksDrawer({
+  collTokenName,
+  troveId,
+}: {
+  collTokenName: string;
+  troveId: bigint;
+}) {
+  return (
+    <div
+      className={css({
+        width: "100%",
+        display: "flex",
+        justifyContent: "left",
+        alignItems: "center",
+        gap: 16,
+        marginTop: -22,
+        height: 60,
+        padding: "12px 16px",
+        whiteSpace: "nowrap",
+        background: "token(colors.fieldSurface)",
+        borderRadius: 8,
+        userSelect: "none",
+      })}
+    >
+      <div
+        className={css({
+          display: "flex",
+          alignItems: "center",
+          gap: 8,
+          marginTop: 18,
+          fontSize: 14,
+        })}
+      >
+        <span>
+          Trove History:
+        </span>
+        {troveExplorers.length > 0 && troveExplorers[0] && (
+          <>
+            <TroveExplorerLink
+              troveExplorer={troveExplorers[0]}
+              collTokenName={collTokenName}
+              troveId={troveId}
+            />
+            {troveExplorers.length > 1 && troveExplorers[1] && (
+              <>
+                {" "}|{" "}
+                <TroveExplorerLink
+                  troveExplorer={troveExplorers[1]}
+                  collTokenName={collTokenName}
+                  troveId={troveId}
+                  last
+                />
+              </>
+            )}
+          </>
+        )}
```

Comment:
When only one Trove explorer is configured, the `last` flag isn't properly set (the external link icon's missing):

<img width="565" height="119" alt="Image" src="https://github.com/user-attachments/assets/22320692-219f-409b-a1b8-ed2fd554bfe2" />